packages/pds-ember/addon/components/pds/textarea/stories/disabled/invalid.stories.js

```diff
@@ -0,0 +1,47 @@
+import hbs from 'htmlbars-inline-precompile';
```

Comment:
I know disabled + invalid is a possible combination, but in practice is it ever likely to be used? Looking at the storybook stories, it might be more obvious that they were invalid if there was text content.

Reply:
If previous experience has shown me, any configuration option that's made available _can_ and _will_ be used in unexpected configurations.  So if a consumer somehow configures the element to be both _logically_ `:disabled` and _visually_ invalid (i.e., `@invalid={{true}} disabled={{true}}`), I want to make sure that the disabled styling wins.  These stories are built specifically to test these scenarios, not to demonstrate proper configuration.

If a consumer takes advantage of the [Constraint Validation API](https://developer.mozilla.org/en-US/docs/Web/API/Constraint_validation) (which is highly recommended), then logically `:invalid` styles will never apply as long as the element is also logically `:disabled`.